INSTALL.md

```diff
@@ -64,7 +65,41 @@ There are several options to install these dependencies depending on your needs:
 - Alternatively, you could install them via conda. However, the conda repository may not have the exact versions of these dependencies that you want.
 - You could setup a full development environment with [Vagrant](https://www.vagrantup.com/), and continue using conda under it to primarily manage python dependencies. Note that we don't cover Vagrant in these instructions.
 
-### PostgreSQL installation on Mac OS X
+### PostgreSQL installation on Linux
+The following instructions have been adapted from [this site](https://computingforgeeks.com/how-to-install-postgresql-13-on-ubuntu/) and tested on Ubuntu v20.04.4 for Postgres v13.
+
+First, ensure that you have updated packages and reboot the system with:
+```bash
+sudo apt update && sudo apt -y full-upgrade
+[ -f /var/run/reboot-required ] && sudo reboot -f
+```
+You can reboot the system with `sudo reboot` in case any packages were updated.
+
+Next, we need to add the Postgres repository to our system:
+```bash
+sudo apt update
+sudo apt install curl gpg gnupg2 software-properties-common apt-transport-https lsb-release ca-certificates
+curl -fsSL https://www.postgresql.org/media/keys/ACCC4CF8.asc|sudo gpg --dearmor -o /etc/apt/trusted.gpg.d/postgresql.gpg
+echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" |sudo tee  /etc/apt/sources.list.d/pgdg.list
+```
+Adding the repository has added many different packages, which allows us to now install Postgres v13 with the following commands:
+```bash
+sudo apt update
+sudo apt install postgresql-13 postgresql-client-13
+```
+Now, we need to reconfigure the `pg_hba.conf` file and change all occurrences of `md5` and `peer` to `trust`. You can access the file with:
+```bash
+sudo vim /etc/postgresql/13/main/pg_hba.conf
+```
+To make sure all changes have been reflected, restart the Postgres server:
+```bash
+sudo service postgresql restart
+```
+Installing Postgres is now complete. Note that you will need to start the Postgres server every time you start the Qiita server. You can do this with the following command:
+```bash
+sudo service postgresql start
+```
+### PostgreSQL installation on Mac OS X (Outdated)
```

Comment:
@aliu104 I'm confused as to why '(outdated)' has been added to this title. As far as I know, the installation instructions are still correct for MacOSX, and if they're not, they should be updated. Instructions marked as 'obsolete' doesn't make much sense, IMHO.

Reply:
@charles-cowart We labelled these instructions as outdated temporarily since the instructions were for Postgres v9.5 and we did not have a Mac to test them. However, we are currently in the process of obtaining a Macbook and testing these instructions and will keep you posted!

Reply:
@aliu104 let's not do temporarily, since these instructions become official as soon as I approve and merge them. Usually the time one would test such things is before issuing the PR. It's all right in this case to change v9.5 below to "v9.5, v13". All Qiita developers in Knight Lab and CMI use MacOSX and developed and tested the code against v9.5 and v13 databases using Postgres.app.